### PR TITLE
[PR] Support lightweight cpm and hourglass

### PR DESCRIPTION
**Pull request by tucan9389**
- Add dataset config file #33 
- Add training config file #33 
- Implement [mv2 cpm](https://github.com/edvardHua/PoseEstimationForMobile/blob/master/training/src/network_mv2_cpm.py) with tf2 #31 
- Implement [mv2 hourglass](https://github.com/edvardHua/PoseEstimationForMobile/blob/master/training/src/network_mv2_hourglass.py) with tf2 #30 
- Measure PCKh with validation dataset  #32 
- Support tensorboard #28
- Restructure folder hierarchy #29 
- Convert to tflite from saved_model #14

### CODE REVIEW

**Comment by tucan9389** [comment]
@idchlife 
Hi, I made a big PR. Could you please review the PR? 
If you have any question, tell me feel free.
Thank you.

**Comment by idchlife** [comment]
@tucan9389 code seems to be ok! Interesting thing that I also started working on moving all paths outside :D 
It's good that you implemented this. Because of work I don't know when I would have time to finish my code 

If code executes I think it would be nice to merge 👍

File: config/dataset/ai_challenger-colab.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[dataset]
+dataset_root_path: 'datasets'
+dataset_directory_name: 'ai_challenger'
+train_images: 'train/images'
+train_annotation: 'train/annotation.json'
+valid_images: 'valid/images'
+valid_annotation: 'valid/annotation.json'
```
Comment (canapio) [inline config/dataset/ai_challenger-colab.cfg:7]: TEST

**Review by canapio — APPROVED** [review]
TEST for approving

**Pull request merged**